Fix typos

Dimitrolito opened the pull request:
This pull request addresses minor typos in the following files:
- `common_test.go`: Corrected the use of "a" to "an" for "Actor" in the `String()` method.
- `ema_scores.go`: Fixed "a active set worker" to "an active set worker" for grammatical accuracy.
- `msg_server_reputer_payload.go`: Corrected "a individual loss" to "an individual loss".
- `msg_server_worker_payload.go`: Fixed "a individual inference" to "an individual inference".

These changes ensure proper grammar usage and improve code clarity.

Dimitrolito committed:
typos common_test.go

Signed-off-by: Dmytrol <46675332+Dimitrolito@users.noreply.github.com>

Dimitrolito committed:
typos ema_scores.go

Signed-off-by: Dmytrol <46675332+Dimitrolito@users.noreply.github.com>

Dimitrolito committed:
typos msg_server_reputer_payload.go

Signed-off-by: Dmytrol <46675332+Dimitrolito@users.noreply.github.com>

Dimitrolito committed:
typos msg_server_worker_payload.go

Signed-off-by: Dmytrol <46675332+Dimitrolito@users.noreply.github.com>

kpeluso commented on test/fuzz/common_test.go:
```diff
@@ -53,7 +53,7 @@ type Actor struct {
 
 // stringer for actor
-func (a Actor) String() string {
+func (an Actor) String() string {
 	return a.name
 }
```
Hey this change would result in a `nil` pointer dereference error

relyt29 replied:
also it makes no sense lol 

a is the name of the actor, not an [article](https://owl.purdue.edu/owl/general_writing/grammar/using_articles.html) describing a particular actor

spooktheducks replied:
actually it wouldn't even compile unless there's an unexported package local variable called `a`

Dimitrolito replied:
the issue has been fixed

Dimitrolito committed:
Update common_test.go

Signed-off-by: Dmytrol <46675332+Dimitrolito@users.noreply.github.com>

Dimitrolito commented:
Hi @kpeluso 

Just a reminder about pull request #715. It fixes typos in the following files:

- `ema_scores.go`: fixed "a active set worker" to "an active set worker" for grammatical accuracy.
- `msg_server_reputer_payload.go`: corrected "a individual loss" to "an individual loss."
- `msg_server_worker_payload.go`: fixed "a individual inference" to "an individual inference."

Thank you!

Dimitrolito commented:
Hi @kpeluso, @relyt29, @spooktheducks

Just a friendly reminder about PR #715: it fixes typos in  
- `ema_scores.go` (“a active set worker” → “an active set worker”)  
- `msg_server_reputer_payload.go` (“a individual loss” → “an individual loss”)  
- `msg_server_worker_payload.go` (“a individual inference” → “an individual inference”)  

Could you please take another look when you have a moment? Thanks! 😊
